### PR TITLE
Seeding the SARIF v2.2 workplace - separating source and deliverables

### PR DESCRIPTION
**Pull request by sthagen**
Still not requesting a merge into the baseline, but mirroring the experimental branch of my fork in the main repo on a feature branch with the same name.
Rationale: Make the exploration more visible to encourage a wider feedback.

The user facing "GFM+gh_cosmetics" delivery item `sarif-2.2/prose/sarif-v2.2-editor-draft.md` (huge single file online rendition of markdown plus HTML work arounds) can be generated already as is from the source files below `sarif-2.2/prose/edit/src/` with the binder definition in `sarif-2.2/prose/edit/etc/bind.txt`, and the proof of concept filter at `sarif-2.2/prose/edit/bin/volatile.py`. 

As of commit `59af4650a` this will render like so: <https://github.com/sthagen/oasis-tcs-sarif-spec/blob/versioned-spec-folders/sarif-2.2/prose/share/sarif-v2.2-draft.md>
> @sthagen will update the above direct link to the rendered "elephant" any time the head moves i.e. this branch is updated.

The history of the refactoring can be best traced at (freshest change on top):

- Sourcehut: https://git.sr.ht/~sthagen/oasis-tcs-sarif-spec/log/versioned-spec-folders
- GitHub: https://github.com/sthagen/oasis-tcs-sarif-spec/commits/versioned-spec-folders

The list of commits below this description in the pull request is ordered as freshest change at the bottom ... but otherwise identical (only more scrolling needed to follow).

**Update**: I started to derive a next generation SARIF base schema from the errata version to implements #580 (cf. [sarif-2.2/schema/sarif-2-2.schema.json](sarif-2.2/schema/sarif-2-2.schema.json)) which will resolve as link after merge. 

**Another update**: I will provide a markdown version of v2.1 (very similar to the current seed candidate for v2.2) in the sarif-2.1 tree (to support the TC per the markdown discussion in meeting no. 81 on 2023-NOV-09 and the corresponding action).

**And ...**: A self-contained single-file HTML version is now available as derived from the Markdown in the `sarif-2.2/prose/share/` folder. One can download (e.g. from <https://git.sr.ht/~sthagen/oasis-tcs-sarif-spec/blob/versioned-spec-folders/sarif-2.2/prose/share/sarif-v2.2-draft.html>) and read the draft in the browser (locally).

### CODE REVIEW

**Review by michaelcfanning — APPROVED** [review]

File: sarif-2.2/prose/edit/src/frontmatter.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+<img src="media/OASISLogo-v3.0.png" style="width:3.01036in;height:0.61978in" />
+
+Static Analysis Results Interchange Format (SARIF) Version 2.1.0 Plus Errata 01 (redlined)
```
Comment (dmk42) [inline sarif-2.2/prose/edit/src/frontmatter.md:3]: We can remove the `(redlined)` notation.

**Review by dmk42 — COMMENTED** [review]
As soon as the Approved Errata document is published, we can check it for any adjustments (such as URL changes) and incorporate them into this text.

When you are ready, we should discuss this in a meeting so people can start getting a look at it.

File: sarif-2.2/prose/edit/src/frontmatter.md (hunk 1 above)
Comment (sthagen) [reply sarif-2.2/prose/edit/src/frontmatter.md:3]: Will do, thanks.

**Comment by sthagen** [comment]
The TC received the starter pack for v2.2 from OASIS administration (<https://lists.oasis-open.org/archives/sarif/202310/msg00004.html>) and stored the archive for future reference at <https://www.oasis-open.org/committees/document.php?document_id=71442&wg_abbrev=sarif>.

I will adapt the sources and generated markdown in the change set of this pull request in the next days as follows:
>Submitter's message
I (Stefan) will use this content to morph our work product documented in https://github.com/oasis-tcs/sarif-spec/pull/599 and when containing all errata 01 fixes into a real SARIF v2.2 initial draft to which we (the editors) then can apply all future (TC) approved changes from the (GitHub) issue proposals.
-- Mr. Stefan Hagen (<https://lists.oasis-open.org/archives/sarif/202310/msg00005.html>)

File: sarif-2.2/prose/edit/src/frontmatter.md (hunk 1 above)
Comment (sthagen) [reply sarif-2.2/prose/edit/src/frontmatter.md]: Now the frontmatter is aligned with the published errata at <https://docs.oasis-open.org/sarif/sarif/v2.1.0/errata01/os/sarif-v2.1.0-errata01-os-complete.html>

**Review by dmk42 — APPROVED** [review]
Thanks for doing this!

The decision to organize this way belongs to the editors, and I am approving on that basis.

**Pull request merged**